GitUI/SplitterManager.cs

```diff
@@ -0,0 +1,192 @@
+﻿using GitUIPluginInterfaces;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public class SplitterManager
+    {
+        private readonly ISettingsSource _settings;
+        private readonly List<SplitterData> splitters = new List<SplitterData>();
+        private float _designTimeFontSize;
+
+        public SplitterManager(ISettingsSource settings, float designTimeFontSize = 8.25F)
```

Comment:
I don't know exactly how it works (because of that I prefer DPI scaling :) ). 
But I found that:
```
AutoScaleDimensions.Height = (int)Font.GetHeight();
AutoScaleDimensions.Width = Font.Size - 1;
```
You can try scale based on Font.GetHeight()